operator/pkg/cli/migrate_v2.go

```diff
@@ -0,0 +1,70 @@
+package cli
+
+import (
+	"fmt"
+	"log"
+
+	ecv1beta1 "github.com/replicatedhq/embedded-cluster/kinds/apis/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/operator/pkg/cli/migratev2"
+	"github.com/replicatedhq/embedded-cluster/operator/pkg/k8sutil"
+	"github.com/spf13/cobra"
+)
+
+// MigrateV2Cmd returns a cobra command for migrating the installation from v1 to v2.
+func MigrateV2Cmd() *cobra.Command {
+	var installationFile, licenseSecret, appVersionLabel string
+
+	var installation *ecv1beta1.Installation
+
+	cmd := &cobra.Command{
+		Use:          "migrate-v2",
```

Comment:
This command will not be used and instead will be called from the upgrade command